src/main/java/io/github/jasmin/member/controller/MemberController.java

```diff
@@ -0,0 +1,36 @@
+package io.github.jasmin.member.controller;
+
+import io.github.jasmin.member.models.MemberInfo;
+import io.github.jasmin.member.models.ReqMember;
+import io.github.jasmin.member.service.MemberService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.List;
+
+@Controller
```

Comment:
```suggestion
@RestController
```
API만 만드는것이었다면 `@ResponseBody`를 포함한 `@RestController` 가 더 좋아보입니다!

Reply:
저도 같은 생각입니다.!👍